### PR TITLE
Create readme and contribution guide for Python projects

### LINKED ISSUE #20: Create Readme template for Python projects

### LINKED ISSUE #21: Follow guidelines for contribution guide
For Java: move contribution guide from README.md to CONTRIBUTING.md as per official guidelines.
For Python: create contribution guide

https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors#adding-a-contributing-file

### PR DESCRIPTION
**Pull request by disrupted**
Closes #20
Related to #21

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+This repository contains templates for open source projects bakdata. This includes baseline configs for language tools, default project dependencies, and example CI setups.
 
-Currently, this repository contains templates for:
- - Java projects with Gradle ([java-gradle](https://github.com/bakdata/bakdata-project-templates/tree/master/java-gradle))
+- [Java projects with Gradle](https://github.com/bakdata/bakdata-project-templates/tree/master/java-gradle)
+- [Python projects with Poetry](https://github.com/bakdata/bakdata-project-templates/tree/master/python-poetry)
 
 ## How to Use It
+
 This repository contains a script that will fetch all templates and guide you through the process of setting up your new project.
```
Comment (disrupted) [inline README.md:10]: I am not sure if this is still useful, but if so we should update it to support initializing Poetry projects as well

Comment (disrupted) [reply README.md:10]: I am getting the error `Command travis required but not installed.` so I assume this hasn't been used in a long time.

Comment (sujuka99) [reply README.md:10]: I doubt that I would ever want to run the script. I'd vote for removing it.

**Review by sujuka99 — DISMISSED** [review]
LGTM

Please consider ~removing or~ updating `init.sh` in a follow up PR

File: templates/python-poetry/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+```sh
+poetry install
+```
+
+3. Install the pre-commit hooks for linting and to enforce a consistent code style.
+
+```sh
+pre-commit install
+```
```
Comment (jkbe) [inline templates/python-poetry/CONTRIBUTING.md:19]: I think these steps don't apply for this repo 😅

File: templates/python-poetry/LICENSE [hunk 3]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) bakdata
```
Comment (jkbe) [inline templates/python-poetry/LICENSE:3]: In [the Java project](https://github.com/bakdata/bakdata-project-templates/blob/master/templates/java-gradle/LICENSE), there's also a copyright year here which seems to get templated somehow: `Copyright (c) {{year}} bakdata`

**Review by jkbe — DISMISSED** [review]

File: templates/python-poetry/LICENSE (hunk 3 above)
Comment (disrupted) [reply templates/python-poetry/LICENSE:3]: hm, I am not a copyright lawyer but I've seen many open source projects abandon the year in their copyright notice/license, so I would suggest we do the same

File: templates/python-poetry/CONTRIBUTING.md (hunk 2 above)
Comment (disrupted) [reply templates/python-poetry/CONTRIBUTING.md:19]: yes, it doesn't apply to this repository. it is not linked in the root readme, only for the Python project. A similar one should be created for the Java template (#21)

File: README.md (hunk 1 above)
Comment (disrupted) [reply README.md:10]: it also handles the replacement of placeholders for the Java template though

Comment (sujuka99) [reply README.md:10]: Ah, nvm then :D

File: templates/python-poetry/CONTRIBUTING.md (hunk 2 above)
Comment (jkbe) [reply templates/python-poetry/CONTRIBUTING.md:19]: Ah, my bad, I overlooked that this file is not in root.

File: templates/python-poetry/LICENSE (hunk 3 above)
Comment (philipp94831) [reply templates/python-poetry/LICENSE:3]: IntelliJ has a feature that automatically updates the year. PyCharm probably as well but I have no strong opinion here

Comment (disrupted) [reply templates/python-poetry/LICENSE:3]: I think the license itself is still valid even without the year in it. Therefore I would skip it

**Review by sujuka99 — APPROVED** [review]

**Pull request merged**